test/test_kmer_collection.cc

```diff
@@ -43,10 +43,60 @@ TEST_CASE("Basic functionalities of kmer_collection", "[kmer_collection]") {
     REQUIRE(itBegin != itEnd);
   }
 
-  SECTION("comparison for equality") {
+  SECTION("Comparison for equality 1") {
     REQUIRE(itBegin == itBegin);
     REQUIRE(itEnd == itEnd);
```

Comment:
Are these `REQUIRE` not [tautological](https://en.wikipedia.org/wiki/Tautology_(logic))?